database/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
 
 # Create your models here.
+class Request(models.Model):
+    request_ID = models.IntegerField(max_length=50)
+    user_access_ID= models.CharField(max_length=250)
+    request_time = models.DateTimeField(max_length=100)
+
+class User(models.Model):
```

Comment:
there are different types of users with different levels of access

Reply:
ung default user model inheritance ba yan or extends na i have to place all the fields?

Reply:
No need to place the fields since yung default User model may email, password and ID na. For the user type, ano ulit yung different types of users natin HAHA

Reply:
admin and adviser